extensions/adobe/experience/adcloud/attributedconversions.schema.json

```diff
@@ -0,0 +1,241 @@
+          "type": "string",
+          "description": "Placement Domain"
+        },
+        "xdm:referrerUrl": {
+          "title": "Referrer URL",
+          "type": "string",
+          "description": "The referrer is the webpage that sends visitors to your site using a link"
+        },
+        "xdm:searchTerm": {
+          "title": "Search Term",
+          "type": "string",
+          "description": "Original Term used for the Search"
+        },
+        "xdm:geoipLocationId": {
+          "title": "Geo IP Location Identifier",
+          "type": "number",
+          "description": "A numeric identifier that is obtained from Maxmind’s GeoIP package that represents the Geo"
+        },
+        "xdm:propertyName": {
+          "title": "Property Name",
+          "type": "string",
+          "description": "Property Name Configured by Pixel, Analytics, Custom Feeds"
+        },
+        "xdm:clickThroughValue": {
+          "title": "Click Through Value",
+          "type": "number",
+          "description": "Raw Conversion Value if Triggering Event Type is Click"
+        },
+        "xdm:clickThroughCount": {
+          "title": "Click Through Count",
+          "type": "number",
+          "description": "Raw Conversion Count if Triggering Event Type is Click"
+        },
+        "xdm:viewThroughValue": {
+          "title": "View Through Value",
+          "type": "number",
+          "description": "Raw Conversion Value if Triggering Event Type is Impression"
+        },
+        "xdm:viewThroughCount": {
+          "title": "View Through Count",
+          "type": "number",
+          "description": "Raw Conversion Count if Triggering Event Type is Impression"
+        },
+        "xdm:clickviewThroughValue": {
+          "title": "Click View Through Value",
+          "type": "number",
+          "description": "Raw conversion value if trigger event is click but impressions present in path"
+        },
+        "xdm:clickviewThroughCount": {
+          "title": "Click View Through Count",
+          "type": "number",
+          "description": "Raw conversion Count if trigger event is click but impressions present in path"
+        },
+        "xdm:viewclickThroughValue": {
+          "title": "View Click Through Value",
+          "type": "number",
+          "description": "Raw conversion value if trigger event is impression but clicks present in path"
+        },
+        "xdm:viewclickThroughCount": {
+          "title": "View View Through Count",
+          "type": "number",
+          "description": "Raw conversion count if trigger event is impression but clicks present in path"
+        },
+        "xdm:pathSize": {
+          "title": "Path Size",
+          "type": "integer",
+          "description": "Path Size of Trigger Event Type. It is a numerical value that represents the count of the conversion path that triggered the event"
+        },
+        "xdm:conversionType": {
+          "title": "Conversion Type",
```

Comment:
Same comment as in `adconversions.schema.json`. Can you use `shortint`, if available ?

Reply:
short will suffice here

Reply:
There is no short type in XDM. https://wiki.corp.adobe.com/display/DMSArchitecture/XDM+Architecture#XDMArchitecture-XDMDataTypes

Reply:
There is not short type in JSON Schema. If you want the JSON Schema fields treated as a short in the AEP system, then you define the field as:
"sampleField": {
  "type":"integer",
  "minimum":-32768,
  "maximum":32767
}

Reply:
@cmathis - Thank you